src/errors/TooManyRequestsError.ts

```diff
@@ -0,0 +1,14 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+// eslint-disable-next-line import/prefer-default-export
+export class TooManyRequestsError extends Error {
```

Comment:
It's likely that this error is not needed. See https://github.com/awslabs/fhir-works-on-aws-routing/pull/8#discussion_r485111330

Reply:
In this case, the persistence layer would be throwing a `TooManyRequestsError` if it finds that there are too many concurrent export jobs. Since it's coming from the persistence layer, did we still want to just use `http-errors`?

Reply:
The idea is that the persistence layer is not aware of http at all. So it can instead throw an error in persistence terms like `ExportJobsLimitExceededError`(just an idea) and then the router can translate it to an http 429 error.

Reply:
fixed